gamestonk_terminal/economy/fred/fred_view.py

```diff
@@ -51,9 +51,73 @@ def notes(series_term: str, num: int):
     print("")
 
 
-def display_series(series: str, start_date: str, raw: bool, export: str):
+def display_fred_series(
+    d_series: Dict, start_date: str, raw: bool = False, export: str = ""
+):
     """Display (multiple) series from https://fred.stlouisfed.org. [Source: FRED]
+    Parameters
+    ----------
+    series : str
+        FRED Series ID from https://fred.stlouisfed.org. For multiple series use: series1,series2,series3
+    start_date : str
+        Starting date (YYYY-MM-DD) of data
+    raw : bool
+        Output only raw data
+    export : str
+        Export data to csv,json,xlsx or png,jpg,pdf,svg file
+    """
+    series_ids = d_series.keys()
+    data = pd.DataFrame()
+
+    for s_id in series_ids:
+        data = pd.concat(
+            [
+                data,
+                pd.DataFrame(
+                    fred_model.get_series_data(s_id, start_date), columns=[s_id]
+                ),
+            ],
+            axis=1,
+        )
+
+    # Try to get everything onto the same 0-10 scale.
+    # To do so, think in scientific notation.  Divide the data by whatever the E would be
+    fig, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
+
+    for s_id, title in d_series.items():
+        data_to_plot = data[s_id].dropna()
+        exponent = int(np.log10(data_to_plot.max()))
+        data_to_plot /= 10 ** exponent
+        multiplier = f"(x{str(10**exponent)})" if exponent > 0 else ""
+        ax.plot(data_to_plot.index, data_to_plot, label=f"{title} {multiplier}")
```

Comment:
The labels here tend to be pretty long. This results in the chart being off screen and not great looking in general
![image](https://user-images.githubusercontent.com/11668535/145198078-d945e028-72cf-4856-9ab6-43f398b4f25e.png)

The solution I've used in the sia menu for long company names is to split the name into multiple lines. I've had a brief look at the series that are available in the menu and I see that the longer ones tent to have a comma in the title. If the title is split to multiple lines using a comma as a delimiter the chart can come out a bit nicer.
```python
title = title.replace(", ", ",\n")
ax.plot(data_to_plot.index, data_to_plot, label=f"{title} {multiplier}")
```
![image](https://user-images.githubusercontent.com/11668535/145198504-347695d5-4740-4041-8d97-61d37ad260d0.png)

This is still not ideal, but "it's beta that nothing"

Reply:
I don't like this.  textwrap is better.  I will play with different widths later and report what works.

Biggest issue is that just makes the chart bigger (tall) which is better than making it super wide.

Reply:
Can you give me a use case example that would produce the most horrific chart so that I try to play with the layout too?

Reply:
So one can just look through the Fred website for long titles.

Example: CUUR0000SA0R

Would anyone ever use this- I have no idea but.  But If they are comparing 5 or 6 CPI things the legend will be 6 tall (similar to the size of the example treasury yields) and splitting 2 or 3 of them make that almost 10 tall.

Reply:
Here is an example with text wrap (I also added what units the data is in from fred)

<img width="800" alt="Screen Shot 2021-12-08 at 9 46 32 AM" src="https://user-images.githubusercontent.com/18151143/145228571-1035130f-86ac-4883-ba2b-d8882e946c28.png">

Reply:
Great! The chart looks a lot better despite the numbers being depressing lol

Reply:
lol.  I just don't like how the scales can lose value with putting everything on the same 0-10 scale

<img width="798" alt="Screen Shot 2021-12-08 at 10 02 04 AM" src="https://user-images.githubusercontent.com/18151143/145231215-1011821e-e58e-4f13-b08d-80bedcb8e3ab.png">

Reply:
I much prefer this one tbh